cfme/tests/infrastructure/test_delete_infra_object.py

```diff
@@ -16,7 +14,7 @@
 ]
 
 
-def test_delete_cluster_appear_after_refresh(setup_provider, provider, appliance):
+def test_delete_cluster_appear_after_refresh(provider, appliance):
```

Comment:
**QUESTION**
Why did you delete `setup_provider`?

Reply:
@quarckster No need of it; What's your opinion?
I think as we are marking `pytest.mark.provider` now no need to go with `setup_provider`.

Reply:
my bad; forgot to add a fixture `pytest.mark.usefixtures('setup_provider')` @quarckster done